### ISSUE TITLE
Not getting it to work with Oled

### ISSUE THREAD

**Issue by Ryukenden88** [EverythingSmartHome/fingerprint-mqtt#12]
Hello,

when i use the first version fingerprint-mqtt.ino the sensor works perfectly.
i followedd your video and it works in home assistant too.

When i try any other version it doesnt. I did solder the extra wires 4 and 5 to D3 and 3v as written.
but non of the codes work with it. I am using a Wemos d1 mini for it with a OLED on pins D1 sda, and D2 scl.

Any idea what could be causing this?

**Comment by Ryukenden88** [comment]
Edit:

Got the screen to work, but now it is stuck on Ready. Nothing happens when i put my finger on the sensor (which is not green now)

**Comment by Ryukenden88** [comment]
Edit 2:

It works now, turns out the code says:
#define SENSOR_TOUT 10                //GPIO Pin for SENSOR T_OUT

But it should be:
#define SENSOR_TOUT 0                //GPIO Pin for SENSOR T_OUT

Pin d3 on the Mini is Gpio 0

Now the next problem is, the functions in home assistant are not working. The learn/delete and Fingerprint status all stopped working. The device DID remember 3 id's that i programmed in before with the original code, i just cant add any new ones either.

**Comment by Ryukenden88** [comment]
Edit 3:

Home assistant functions did stop, but i managed to add new fingerprints to it with MQTT with the fingerprint_sensor/1/request topic.
Now the next thing is, can you add a name or is it just id 1, id 2 etc?

**Comment by EverythingSmartHome** [comment]
Hello, I see you've made progress. I didn't not write the code for the OLED sketch or the disabling the LED function hence the pins used by someone else may have been different - I haven't tested it.

To get the learn script to function correctly, remove the first action inside the script and it will work after that for learning within HA.

No, this fingerprint sensor does not support names, only IDs, names have to be done within HA.

**Comment by Ryukenden88** [comment]
Hello,

thank you for your reply.
im not sure which part includes the fist action, do you mean remove all of this?

```
  - service: input_text.set_value
    data_template:
      entity_id: input_text.fingerprint_data
      value: |-
        {% if states('input_text.fingerprint_data') == "unknown" %}
          {{"[]"}}
        {% endif %}

```
Alright, no names is fine i can live with that.

**Comment by EverythingSmartHome** [comment]
Yes that's correct, removing that should allow it to work. Jump into the discord if you need some more help

**Comment by Ryukenden88** [comment]
Thank you!

i swapped it out with the oled script and the learn/delete butons work again.
The only part now not working is the  Fingerprint status glance card, which includes Fingerprint Sensor mode, Fingerprint Status Mode and Fingerprint Person.

Ill drop by later today as im off to work now.
Thank you yet again for your help!
